app/jobs/delete_missed_reservations_job.rb

```diff
@@ -0,0 +1,36 @@
+class DeleteMissedReservationsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform
+    return if disabled
+    run
```

Comment:
okay I totally forgot to rename the methods, will do that in the next refactoring update after we decide how much we want to take out of the individual email jobs